modules/samples-operator-configuration.adoc

```diff
@@ -55,22 +45,22 @@ string or registry.redhat.io.
 is supported.
```

Comment:
|`architectures` 
|Placeholder to choose an architecture type. Currently only x86 is supported.

"Currently only x86 is supported."  need removed.